BUILDING.md

```diff
@@ -11,8 +13,8 @@ Note: Use `cmake -G "Visual Studio 15 2017 Win64" ..` as the third command for a
 
 Under Windows the resulting EPANET toolkit library `epanet2.dll` and its command line executable `runepanet.exe` will be in the `build\bin\Release` directory. The `build\lib\Release` directory contains an `epanet2.lib` file which is needed to build C/C++ applications using the Windows version of the library. For Linux and Mac OS the EPANET toolkit shared library `libepanet2.so` appears in the `build/lib` directory and the command line executable `runepanet` is in the `build/bin` directory. 
 
-In addition, two Windows one-click-build scripts are included in the `win_build` directory:
-1. `Makefile2.bat`: this script uses the `CMake` file and requires the build tools for Visual Studio available from [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/). The Community version will work just fine. This script was tested with Visual Studio 2017 and 2019.
-2. `Makefile.bat`: this is the legacy build script compatible with Visual Studio 2010 which conforms with the C89 Standard which was the standard EPANET supported from earlier versions. This script requires the installation of Microsoft Windows SDK 7.1 ([https://www.microsoft.com/en-us/download/details.aspx?id=8279](https://www.microsoft.com/en-us/download/details.aspx?id=8279)).
+As an alternative to using `CMake` for a Windows build, two one-click-build scripts are included in the `win_build` directory:
+1. `Makefile2.bat`: this script requires both `CMake` and the build tools for Visual Studio (available from [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/) be installed on your machine. It can be executed from any command prompt window. The Community version of Visual Studio will work just fine. This script was tested with both Visual Studio 2017 and 2019.
+2. `Makefile.bat`: this build script uses only Visual Studio commands and does not require `CMake`. It is  compatible with both Visual Studio 2010 and 2017 and should be executed from a Visual Studio x86 Native Tools command prompt window.
```

Comment:
The `Makefile.bat` script can be executed from any command prompt window as well. The script will open the VS x86 Native tools command prompt window.

Reply:
It doesn't work on a plain old command prompt window on my machine. I have to run it in a VS developer command prompt window as per [Microsoft's instructions](https://docs.microsoft.com/en-us/cpp/build/walkthrough-compiling-a-native-cpp-program-on-the-command-line?view=vs-2019).

P.S. I should have used the term "Visual Studio developer command prompt" instead of "Visual Studio x86 Native Tools command prompt" in the instructions for Makefile.bat. I will make the change once we all agree on things.

Reply:
OK, I see what you mean. I just double click the batch file and it runs (Windows 10). I don't open the command prompt.

Reply:
Double clicking doesn't work for me. Perhaps you are running VS 2010 which maybe sets up the proper environment to run the Visual C++ compiler from any command prompt window when you install it (I no longer have VS 2010 on my machine so I'm just guessing here). If that's the case maybe we should qualify the instruction by saying:

_"It is compatible with Visual Studio 2010 and higher. For Visual Studio 2015 and higher it must be executed from a Visual Studio native tools or developer command prompt window."_

Reply:
I have VS 2017 installed but do have the Microsoft Windows SDK 7.1 ([https://www.microsoft.com/en-us/download/details.aspx?id=8279](https://www.microsoft.com/en-us/download/details.aspx?id=8279)) so maybe installing it have set the things up. But saying "_...Visual Studio native tools or developer command prompt window_" is on the safe side so let us keep that.

Reply:
I just discovered that under VS 2010 you have to have run `vcvarsall.bat` to set up your environment to execute `cl` from any command prompt window. Otherwise, like with the later versions of VS, you have to use a Visual Studio command prompt window. So now I'm thinking the best way to phrase the instruction is simply:

_"It is compatible with Visual Studio 2010 and higher and should be executed from a Visual Studio command prompt window."_

I'm using the generic term "Visual Studio command prompt window" since it goes by different names in the different versions.